distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -33,3 +33,8 @@ This can be the path to a file or a directory containing the desired files.
 ====
 On Windows, put the file path in quotes: `"path/to/file"`.
 ====
+
+Successful command line ingest operations will be accompanied with messaging indicating how many files were ingested and how long the operations took.
```

Comment:
Use present tense "are" instead of "will be". 
Sentence feels long. Maybe:

"The ingest command prints how many files are ingested and how long the command ran."